package.json

```diff
@@ -16,16 +16,17 @@
   "scripts": {
     "styles": "node_modules/.bin/node-sass data/recommendation --output data/recommendation --sourcemap=none",
     "build": "npm run styles && jpm xpi",
-    "lint": "eslint lib data"
+    "lint": "eslint lib data",
+    "test": "jpm test"
   },
   "devDependencies": {
-    "jpm": "1.0.7",
     "eslint": "2.13.1",
     "eslint-config-airbnb": "9.0.1",
     "eslint-plugin-import": "1.10.2",
     "eslint-plugin-jsx-a11y": "1.5.5",
     "eslint-plugin-mozilla": "0.0.3",
     "eslint-plugin-react": "5.2.2",
+    "jpm": "1.1.4",
```

Comment:
This update of jpm changes the output .xpi file's name, so you'll have to update the CI to account for that. I recommend leaving this change for another PR if possible.